packages/components/src/Enumeration/Header/HeaderInput.component.js

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import Action from '../../Actions/Action';
 import theme from './Header.scss';
 
-let inputRef;
+let internalInputRef;
```

Comment:
if this component get used more than once into another component this value will be shared between them.